statsmodels/stats/outliers_influence.py

```diff
@@ -699,7 +699,8 @@ def summary_table(res, alpha=0.05):
 
 
     #standard error for predicted observation
-    predict_se, predict_ci_low, predict_ci_upp = wls_prediction_std(res)
+    (predict_se, predict_ci_low, 
+     predict_ci_upp) = wls_prediction_std(res, alpha=alpha)
```

Comment:
In general I don't like having to read left hand side of assignment over several lines.

two lines with temporary looks better, unless maybe in a heavy loop

```
tmp = wls_prediction_std(res, alpha=alpha)
predict_se, predict_ci_low, predict_ci_upp = tmp
```